jsonnet/prometheus-user-workload.jsonnet

```diff
@@ -139,6 +139,13 @@ local alertmanagerRole =
         },
       },
 
+    // TODO: Adding this to our stack is not as easy
+    // as sidecar listens on 127.0.0.1:10902 and we 
+    // need kube-rbac-proxy in front of it.
```

Comment:
just as a verification point: we have https://issues.redhat.com/browse/MON-1064 that tracks this effort.

Reply:
Yes, see discussion here https://github.com/openshift/cluster-monitoring-operator/pull/960#discussion_r504539782